FunctionalTableData/TableView/FunctionalTableData.swift

```diff
@@ -153,6 +157,15 @@ public class FunctionalTableData: NSObject {
 		tableView?.delegate = nil
 	}
 	
+	/// All rows that are currently visible in the table.
+	public var visibleRows: [CellConfigType] {
+		guard let visibleIndexPaths = tableView?.indexPathsForVisibleRows else {
+			return []
+		}
+		
+		return visibleIndexPaths.compactMap { sections[$0] }
+	}
```

Comment:
What value are you getting out of the CellConfigType? are you then casting those to their actual types? I think this bit property be outside of the library tho